include/lemlib/devices/gyro/gyro.hpp

```diff
@@ -40,41 +42,41 @@ class Gyro {
          *
          * @note 0 is in the positive x direction, and heading increases counterclockwise
          *
-         * @return float heading, in radians, locked from -pi to +pi
+         * @return Angle heading, locked from -pi to +pi
```

Comment:
```suggestion
         * @return Angle heading, locked from west to east
```
Remove radian-specific units